Solutions/EntrustIdentityAsaService/Playbooks/EntrustPlaybooks/Entrust-EnrichIncidentWithIPDetails/readme.md

```diff
@@ -0,0 +1,61 @@
+# Entrust-Enrich Incident with IP Details Playbook
+ ## Summary
+ When a new Microsoft Sentinel incident is created, this playbook gets triggered and performs below actions
+ 1. Fetches the IP details of his authentication and management activities from Entrust
+ 2. Adds a rich comment to the incident with all the collected information
```

Comment:
Description should be around IP entity, but it is around name entity.

Reply:
Done